action-server/covidflow/utils/maps.py

```diff
@@ -0,0 +1,38 @@
+import base64
+import hashlib
+import hmac
+import os
+from typing import Tuple
+from urllib.parse import urlencode
+
+GOOGLE_MAPS_SIZE_ENV = "GOOGLE_MAPS_SIZE"
+GOOGLE_API_KEY_ENV = "GOOGLE_MAPS_API_KEY"
+GOOGLE_SIGN_SECRET_ENV = "GOOGLE_MAPS_URL_SIGN_SECRET"
+
+MAPS_ENDPOINT = "https://maps.googleapis.com"
+MAPS_API_PATH = "/maps/api/staticmap"
+
+
+def get_map_url(coordinates: Tuple[float, float]):
+    key = os.environ[GOOGLE_API_KEY_ENV]
+    secret = os.environ[GOOGLE_SIGN_SECRET_ENV]
+    size = os.environ.get(GOOGLE_MAPS_SIZE_ENV, "220x150")
```

Comment:
Not sure about using an env variable for this. What about having a function argument for this with a default value we could change if we need to?